plugins/modules/user.py

```diff
@@ -515,9 +530,11 @@ def main():
     module = ForemanUserModule(
         foreman_spec=dict(
             login=dict(required=True, aliases=['name']),
+            disabled=dict(required=False, type='bool', default=False),
```

Comment:
```suggestion
            disabled=dict(required=False, type='bool'),
```

Please don't set defaults unless really necessary. Ideally we depend on the API to provide those defaults to us.